src/model.coffee

```diff
@@ -297,5 +314,11 @@ class Model extends Backbone.Model
   _onAssociatedCollectionChange: (field, collectionChangeDetails) =>
     @attributes[@constructor.associationDetails(field).key] = collectionChangeDetails[1].pluck('id')
 
+  _collectionHasMany: (associator) ->
+    _.isArray(associator) &&
+    _.find(associator, (collectionName) => inflection.singularize(collectionName) == @className())
```

Comment:
If you wanted to drop this fat arrow, you can pass `this` as the final argument to this method.